src/aap_eda/core/tasking/__init__.py

```diff
@@ -40,6 +45,62 @@
     None,
 ]
 
+try:
+    redis_prefix = f"EDA{service_id()[0:8]}-rq"
+except Exception:
```

Comment:
We need here to be cautious. I don't think catching all the exception is a good idea. If it fails by some other unexpected reason, we would generate a new UUID every restart with catastrophic results. I think we should catch only the expected exception: ProgrammingError
Even in that case, we might want to use a fixed value, so it would not change between restarts, would be better to debug and would work correctly. 


But I even in that case, I wonder if it would be better if we can figure out a way detect of migrations are applied instead of catching the exception.